kimchi/src/circuits/gate.rs

```diff
@@ -137,18 +134,9 @@ pub enum CircuitGateError {
     /// Disconnected wires
     #[error("Invalid {typ:?} copy constraint: {},{} -> {},{}", .src.row, .src.col, .dst.row, .dst.col)]
     CopyConstraint { typ: GateType, src: Wire, dst: Wire },
-    /// Invalid copy constraint
-    #[error("Invalid {0:?} copy constraint")]
-    InvalidCopyConstraint(GateType),
-    /// Invalid lookup constraint - sorted evaluations
```

Comment:
Oh, doesn't this ever happen? I bet I once got some error regarding sorted evaluations... but perhaps it was back when coefficients were not fully done

Reply:
That's an old error that's not used any more.  There's a new copy constraint one that's better and tells you what wires failed.